lib/package/package.mk

```diff
@@ -30,7 +30,7 @@
  *	:LICENSE: license-class-pattern
  *	:CATEGORY: category-id ...
  *	:COVERS: package ...
- *	:REQURES: package ...
```

Comment:
Hmm, was this being ignored previously?

Reply:
It must have been. Since the non-typo `:REQUIRES:` version is used elsewhere, I don't think this typo version could have been intended.

All the same, I can take out these changes and make a separate pull request of them, if that is preferred.

Reply:
Actually this is fine as it's only in a comment.